governance-with-multisig/README.md

```diff
@@ -0,0 +1,136 @@
+
+## About this guide
+
+This guide will take you through the following steps:
+
+1- Set up a new ZeppelinOS project
+2- Register a contract and deploy it to a network
+3- Create a new instance of your contract
+4- Create a multisig wallet and transfer upgradeability rights of the new contract instance to it
+5- Register a new version of your logic contract and deploy it
+6- Submit a request to the multisig wallet to upgrade the contract instance
+7- Confirm the request and check the contract was upgraded
+
+## Step by step
+
+We'll begin by setting up the ZeppelinOS project. Clone this repository, install dependencies, and initialize the `zos` project to get started.
+
+```bash
+$ git clone https://github.com/zeppelinos/labs.git
+$ cd labs/governance-with-multisig
+$ npm install
+$ npx zos init governance-with-multisig
+```
+
+Our first step will be to register the `EthBox` contract in our project, deploy the logic contract to the blockchain, and create a new instance. We'll be working on a local ganache instance (you can start one via `./ganache-sh`), though you can also run this on any real network.
+
+```sh
+$ npx zos add EthBox
+$ npx zos push --skip-compile --network local
+$ npx zos create EthBox --network local
+```
+
+We can test that the EthBox instance was correctly deployed by `deposit`ing some ETH on it from a `truffle console`. Replace `MY_ETHBOX_ADDRESS` with the address returned by the last command run above.
+```js
+truffle(local)> EthBox.at(MY_ETHBOX_ADDRESS).deposit({ from: owner1, value: 1e18 })
+```
+
+Up to this point, the `admin` of the `EthBox` instance we have just created is a ZeppelinOS `App`, which centralizes control of our project. This App, in turn, is controlled by the account we used from the CLI. This means that this single account has absolute control to register new logic contracts, create new instances, and upgrade any of the existing ones.
+
+We will now create a multisig wallet, that will hold the keys to the upgreadeability of our new instance of `EthBox`. You can either create the wallet programatically from a `truffle console`, or use the Gnosis provided GUI, downloadeable from [here](https://github.com/gnosis/MultiSigWallet/releases). Take note of the multisig contract address.
+
+```js
+truffle(local)> MultiSigWallet.new([owner1, owner2, owner3], 2, { from: owner1 }).then(i => multisig = i)
+```
+
+To transfer ownership of the `EthBox` instance to the multisig, you can use the `change-admin` script provided here, by specifying the address of the multisig.
+
+```sh
+$ npx truffle exec change-admin.js EthBox $MULTISIG_ADDRESS --network local
```

Comment:
do we need to export the `MULTISIG_ADDRESS` before?

Reply:
Changed for an instruction to "replace it with the actual address"